src/Command/DeleteClientCommand.php

```diff
@@ -31,6 +32,16 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return self::INVALID_ARGUMENT;
         }
         $response = $httpClient->request('DELETE', $url.'/api/clients/'.$id, ['auth_basic' => [$username, $password],]);
-        return $this->returnCode($response, $output);
+        try {
+            $status = $response->getStatusCode();
+        } catch (TransportException $e) {
+            $output->writeln($e->getMessage());
+            return self::ERROR;
```

Comment:
We should use an error code specific for transport exception (communication error)